### PR TITLE
Initial sync from aeternity repo

### PR DESCRIPTION
**Pull request by dincho**
- configuration copy from [aeternity docker-compose](https://github.com/aeternity/aeternity/blob/master/docker-compose.yml) and [docker config files](https://github.com/aeternity/aeternity/tree/master/docker)
- readme copy from [aeternity localnet](https://github.com/aeternity/aeternity/blob/master/docs/docker.md#localnet)
- config files re-organization
- fix state channels websocket API configuration
- improve health checks (legacy internal route and SC WS)

After this initial sync this repo should be tagged as v1.0.0

Related: https://github.com/aeternity/aeternity/pull/2635
[PT Ticket](https://www.pivotaltracker.com/story/show/165181713)

### CODE REVIEW

**Comment by dincho** [comment]
> From a quick look it looks fine though I would like to see corresponding PR in aeternity/aeternity at the same, so to see doc updates and "spot the difference".

here it is: https://github.com/aeternity/aeternity/pull/2635

File: test/healthcheck.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# As this script might be used as docker health check it should exit with either 0/1
```
Comment (lucafavatella) [inline test/healthcheck.sh]: This script seems duplicated in other repo https://github.com/aeternity/aeternity/pull/2635/files

Would it be possible to at least comment in one of the two which one the canonical script is?

Comment (venimus) [reply test/healthcheck.sh]: do you mean https://github.com/aeternity/aeternity/blob/165181713_external_localnet/docker/healthcheck.sh

Comment (dincho) [reply test/healthcheck.sh]: It's different content/script now, this repo holds an extended version and it's used to check this repo only nodes.

The script in the aeternity repo is used to be embedded in the [docker image as `HEALTHCHECK`](https://github.com/aeternity/aeternity/blob/165181713_external_localnet/Dockerfile#L45)

**Pull request merged**